src/content/administration/cleanup.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Cluster Cleanup
```

Comment:
@huguestennier is it possible to use <TierList> is the title of a page? (to mark that the entire feature only applies to a certain tier?)